src/libs/ReportActionsUtils.ts

```diff
@@ -609,6 +622,60 @@ function isConsecutiveActionMadeByPreviousActor(reportActions: ReportAction[] |
     return currentAction.actorAccountID === previousAction.actorAccountID;
 }
 
+// Todo - combine this with `isConsecutiveActionMadeByPreviousActor` so as to not duplicate logic
```

Comment:
Are we planning to do this in a follow up?

Reply:
yeah wanted to push the PR for review faster